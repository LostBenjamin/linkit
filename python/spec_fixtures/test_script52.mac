
%include test_script52_include

# (why is this getting referenced?)

[macro]
    red

