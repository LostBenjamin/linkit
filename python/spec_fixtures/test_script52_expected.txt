4:set:red
5:set:blu

